clang/test/CodeGenSYCL/reqd-sub-group-size-host.cpp

```diff
@@ -0,0 +1,49 @@
+// RUN: %clang_cc1 -fsycl -fsycl-is-host -triple spir64 -disable-llvm-passes %s -emit-llvm -o -  | FileCheck %s
```

Comment:
Not sure we should compile host with spir triple. It is not representative of the real world.

Reply:
@turinevgeny, Thanks for looking at the patch. I have updated triple on host.